koji_containerbuild/plugins/builder_containerbuild.py

```diff
@@ -735,7 +736,11 @@ def createContainer(self, src=None, target_info=None, arches=None,
             self.logger.debug("Starting %s with params: '%s",
                               create_method, orchestrator_create_build_args)
             build_response = create_method(**orchestrator_create_build_args)
-        except (AttributeError, OsbsOrchestratorNotEnabled):
+        except (AttributeError, OsbsOrchestratorNotEnabled,
```

Comment:
Can we have separate except clause below?

Reply:
ACK